src/components/v1/FundingCycle/FundingCyclePreview.tsx

```diff
@@ -17,7 +20,7 @@ export default function FundingCyclePreview({
   return (
     <FundingCycleDetailsCard
       fundingCycleDetails={<FundingCycleDetails fundingCycle={fundingCycle} />}
-      fundingCycleDuration={fundingCycle.duration}
+      fundingCycleDurationSeconds={fundingCycle.duration.mul(SECONDS_IN_DAY)}
```

Comment:
V1 duration is in days, so here, we convert it to seconds.

Reply:
**Praise**